examples/verifying_mfd.py

```diff
@@ -1,16 +1,183 @@
 import desbordante
+import pandas as pd
+from operator import attrgetter
 
-TABLE = 'examples/datasets/theatres_mfd.csv'
-METRIC = 'euclidean'
-LHS_INDICES = [0]
-RHS_INDICES = [2]
-PARAMETER = 5
-
-algo = desbordante.mfd_verification.algorithms.Default()
-algo.load_data(table=(TABLE, ',', True))
-algo.execute(lhs_indices=LHS_INDICES, metric=METRIC,
-             parameter=PARAMETER, rhs_indices=RHS_INDICES)
-if algo.mfd_holds():
-    print('MFD holds')
-else:
-    print('MFD does not hold')
+GREEN_CODE = "\033[1;42m"
+RED_CODE = "\033[1;41m"
+BLUE_CODE = "\033[1;46m"
+DEFAULT_COLOR_CODE = "\033[1;49m"
+
+def print_results(verifier, data, parameter):
+    highlights_list = verifier.get_highlights()
+    if verifier.mfd_holds():
+        print(GREEN_CODE, "MFD holds", DEFAULT_COLOR_CODE)
+    else:
+        print(RED_CODE, "MFD does not hold due to the following items:",
+              DEFAULT_COLOR_CODE)
+        for ind, cluster in enumerate(highlights_list, start=1):
+            print(f"{BLUE_CODE} #{ind} cluster: {DEFAULT_COLOR_CODE}")
+            listPandas = list(map(lambda x: x.data_index, list(cluster)))
+            print(data.iloc[listPandas])
+            max_distance = max(cluster, key=attrgetter("max_distance")
+                               ).max_distance
+            print(f"Max distance: {max_distance} > {parameter}")
+        print()
+
+# Method that showcases distance between coordinates
+def mfd_L2(table="examples/datasets/addresses_coordinates.csv"):
```

Comment:
If you are referring to the text displayed for users, then it could be noted that it does not convey information on where the file is actually located

As for L2, there's a paragraph in PEP8 that states:
"However, know when to be inconsistent – sometimes style guide recommendations just aren’t applicable. When in doubt, use your best judgment"

l2 and L2 are different metrics, and making it lowercase CAN cause confusion

Fixed